app/views/missing_check_in_mailer/missing_check_in.html.erb

```diff
@@ -5,7 +5,7 @@
   </head>
   <body>
     <p>
-      Psssstt! Remember to <%= link_to "check-in for #{@friendly_date}", new_check_in_url(autofill_date: @friendly_date) %>
+      Psssstt! Remember to <%= link_to "check-in for #{@friendly_date}", new_check_in_url(check_in: { worked_at: @friendly_date }) %>
```

Comment:
Yep! The only difference between what you and I did is that the hash I'm passing in mirrors the hash that is defined in the check_in_params method.